modules/calib3d/src/calibration.cpp

```diff
@@ -3219,6 +3221,9 @@ void cv::Rodrigues(InputArray _src, OutputArray _dst, OutputArray _jacobian)
     CV_INSTRUMENT_REGION();
 
     Mat src = _src.getMat();
+    CV_Check(src.rows, (src.rows == 1 && src.cols == 3) || (src.rows == 3 && src.cols == 1) || (src.rows == 3 && src.cols == 3),
```

Comment:
It is
```
#define CV_Check(v, test_expr, msg)
```
Here, somehow it would have been better to have multiple `v` to print the values of `src.rows` and `src.cols` or have a version without `v` maybe: `#define CV_Check(test_expr, msg)`?

Alternative is to use `CV_Assert((src.rows == 1 && src.cols == 3) || (src.rows == 3 && src.cols == 1) || (src.rows == 3 && src.cols == 3));` 

But I think it is better to have an error message and I hope that most of the users will actually read the full message error.